### PR TITLE
https://issues.redhat.com/browse/ACM-12628 Known issue

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/release_notes/known_issues.adoc [hunk 1]
```diff
@@ -31,6 +31,15 @@ Review the known issues for cluster lifecycle with {mce-short}. The following li
 
 Cluster lifecycle known issues and limitations are part of the Cluster lifecycle with {mce-short} documentation.
 
+[#vsphere_deprovis_fails_when_folder_used]
+=== Deprovisioning vSphere cluster might fail when specific folder is used on {ocp-short} version 4.16.
+//2.10:12371
+
+Deprovisioning a VMware vSphere cluster might fail on {ocp-short} version 4.16 when multiple VMware vSphere clusters are in the same folder. See **ocp-link or kcs link** for more information. 
+
```
Comment (swopebe) [inline clusters/release_notes/known_issues.adoc:39]: Should only be pointing to and creating ACM and MCE known issues here, as we decided. I added OCP known issues link to intro for that separation.

File: clusters/release_notes/known_issues.adoc [hunk 2]
```diff
@@ -31,6 +31,15 @@ Review the known issues for cluster lifecycle with {mce-short}. The following li
 
 Cluster lifecycle known issues and limitations are part of the Cluster lifecycle with {mce-short} documentation.
 
+[#vsphere_deprovis_fails_when_folder_used]
```
Comment (swopebe) [inline clusters/release_notes/known_issues.adoc:34]: concerned raised over slack about veering from strategy to keep OCP known issues and ACM known issues separate.

Comment (swopebe) [reply clusters/release_notes/known_issues.adoc:34]: Seems this is an OCP fix? https://issues.redhat.com/browse/ACM-12371

Comment (swopebe) [inline clusters/release_notes/known_issues.adoc:34]: We have been working hard to clean up unnecessary longer anchors and filenames in certain areas of the doc. Also we use dashes here, see the rest of the doc for examples. Please cut this down to make life easier? We really need to think and move towards simplification in the doc.

**Comment by cadawson** [comment]
This was discussed and decided that this is not the right place for it. Closing.

**Pull request closed without merging**